File: LinearSearch.py
# returns string representation of value(s) if value is found
# else returns a blank string if nothing is found
class LinearSearch:
    def search(array, searchvalue):
        values = ""
        for j in array:
            if j is searchvalue:
                values += str(searchvalue)
        return values

    testArray = [0, 2, 4, 6, 8]
    result = search(testArray, 4)
    print(result)

File: LinearSearch.java
// Returns a string representation of value(s) if value is found.
public class LinearSearch {
	
	public static String values = "";
	
	public static String search(int[] array, int searchValue) {
		for(int i = 0; i < array.length; i++) {
			if(array[i] == searchValue) {
				values += searchValue;
			}
		}
		return values;
	}
	
	public static void main(String[] args) {
		int[] testArray = {0, 2, 4, 6, 8};
		String result = search(testArray, 4);
		System.out.println(result);
	}
}
